### PR TITLE
Add tasmota documentation

### PR DESCRIPTION
**Pull request by emontnemery**
## Proposed change

Add documentation for Tasmota integration

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [x] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [x] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
  - https://github.com/home-assistant/core/pull/39624
  - https://github.com/home-assistant/core/pull/41380
  - https://github.com/home-assistant/core/pull/41483
- Link to parent pull request in the Brands repository: https://github.com/home-assistant/brands/pull/1868

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/tasmota.markdown [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+## Configuration
+
+This integration can be configured using the integrations in the
+Home Assistant frontend.
+
+Menu: **Configuration** -> **Integrations**.
+
+Click on the `+` sign to add an integration and click on **Tasmota (beta)**.
+After completing the configuration flow, the Tasmota integration will be
+available.
```
Comment (effelle) [inline source/_integrations/tasmota.markdown:43]: Erik I think is better clarify for now is just for Relays if you don't have finished the setup for lights and sensors yet.

Comment (emontnemery) [reply source/_integrations/tasmota.markdown:43]: Good point, updated like this:
```
## Limitations

- Only relays are supported. Relays will be added as Home Assistant switch entities.
```

**Review by effelle — DISMISSED** [review]
Perfect!

**Review by frenck — APPROVED** [review]
Thanks, @emontnemery 👍

**Pull request merged**